projects/editions/examples/dfm.yml

```diff
@@ -0,0 +1,101 @@
+kernel:
+  image: "linuxkit/kernel:4.9.x"
+  cmdline: "console=ttyS0 console=tty0 page_poison=1"
```

Comment:
I dont think you want `console=tty0` on hyperkit, it will end up discarding a bunch of messages from the console